lib/ansible/modules/cloud/online/online_server_install.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: online_server_install
+short_description: Install an Online server
+description:
+  - Gather facts about the servers.
+  - U(https://www.online.net/en/dedicated-server)
+version_added: "2.8"
+author:
+  - "Ludovic Petetin (@broferek)"
+extends_documentation_fragment: online
+'''
+
+EXAMPLES = r'''
+- name: Install Centos on the server 5824 with the user foo created
+  online_server_install:
+    api_token: '0d1627e8-bbf0-44c5-a46f-5c4d3aef033f'
+    host: "5824"
```

Comment:
Why is the host a number?

Reply:
Because the API only servers by their ID, what would you put instead?

Reply:
Online_server_facts is quite long to gather all hosts facts whereas in my case, I only want one server facts.

If we could filter facts for a specific host then it could be a solution. Without it, I don't think it is a good idea to use it for people with large inventories.

Reply:
Online_server_facts is quite long to gather all hosts facts whereas in my case, I only want one server facts.

If we could filter facts for a specific host then it could be a solution. Without it, I don't think it is a good idea to use it for people with large inventories.

Reply:
Online_server_facts is quite long to gather all hosts facts whereas in my case, I only want one server facts.

If we could filter facts for a specific host then it could be a solution. Without it, I don't think it is a good idea to use it for people with large inventories.